settings.gradle

```diff
@@ -1,3 +1,16 @@
+// https://docs.gradle.com/enterprise/gradle-plugin/
+plugins {
+    id "com.gradle.enterprise" version "3.11.4"
+}
+
+gradleEnterprise {
```

Comment:
❌ I don't want us to use this on CI. We certainly don't want everyone who builds the app to upload data by default!

Reply:
I'd rather not use this at all.